breeze-for-qubes/breeze-for-qubes.spec

```diff
@@ -0,0 +1,309 @@
+%if 0%{?qubes_builder}
+%define _sourcedir %(pwd)/breeze-for-qubes
+%define _builddir %(pwd)/breeze-for-qubes
+%define breeze_epoch %(cat epoch_breeze)
+%define breeze_version %(cat version_breeze)
+%else
+%define breeze_epoch 0
+%define breeze_version 5.5.4
+%endif
+
+%global         base_name   breeze
+
+%global         build_kde4  1
+
+Name:    plasma-breeze
+Version: %{breeze_version}
+Epoch:   %{breeze_epoch}
+Release: 1%{?dist}
+Summary: Artwork, styles and assets for the Breeze visual style for the Plasma Desktop
+
+License: GPLv2+
+URL:     https://qubes-os.org
+
+%global revision %(echo %{version} | cut -d. -f3)
+%if %{revision} >= 50
+%global stable unstable
+%else
+%global stable stable
+%endif
+Source0: http://download.kde.org/%{stable}/plasma/%{version}/%{base_name}-%{version}.tar.xz
+Patch0:  0001-Initial-work-on-QubesOS-support.patch
+
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:  gcc-c++
+BuildRequires:  cmake
+BuildRequires:  kf5-rpm-macros
+BuildRequires:  extra-cmake-modules
+BuildRequires:  qt5-qtbase-devel
+BuildRequires:  qt5-qtx11extras-devel
+
+BuildRequires:  kf5-kservice-devel
+BuildRequires:  kf5-kcmutils-devel
+BuildRequires:  kf5-plasma-devel
+
+# kde4breeze
+BuildRequires:  kf5-kcoreaddons-devel
+BuildRequires:  kf5-kconfig-devel
+BuildRequires:  kf5-kguiaddons-devel
+
+# kstyle
+BuildRequires:  kf5-ki18n-devel
+BuildRequires:  kf5-kcompletion-devel
+BuildRequires:  kf5-frameworkintegration-devel
+BuildRequires:  kf5-kwindowsystem-devel
+BuildRequires:  kdecoration-devel
+
+BuildRequires:  libxcb-devel
+
+BuildRequires:  gettext
+
+# icon optimizations
+BuildRequires:  hardlink
+# for optimizegraphics
+BuildRequires:  kde-dev-scripts
+BuildRequires:  time
+
+Requires:       kf5-filesystem
+
+Requires:       %{name}-common = %{epoch}:%{version}-%{release}
+
+%description
+%{summary}.
+
+%package        common
+Summary:        Common files shared between KDE 4 and Plasma 5 versions of the Breeze style
+BuildArch:      noarch
+%description    common
+%{summary}.
+
+%package -n     breeze-cursor-theme
```

Comment:
Then (after renaming the package), you probably can drop other subpackages (and remove their files in %install). For example I see no reason for overriding cursor theme.